src/configure/configure.ts

```diff
@@ -166,29 +151,43 @@ class Orchestrator {
         if (this.continueOrchestration) {
             await this.getSourceRepositoryDetails();
             await this.getAzureSession();
-            await this.getGithubPatToken();
-            var targetType = await this.getSelectedPipeline();
-            if (!resourceNode) {
-                let selectedResource = await this.getAzureResource(targetType);
-                if (!this.continueOrchestration) {
+            let repoAnalysisResult = await this.getRepositoryAnalysis();
+            let targetType = await this.getSelectedPipeline(repoAnalysisResult);
```

Comment:
looks wierd that `getSelectedPipeline` returns targetType. Rather it should return void and there should be another function like `getSelectedPipelineTargetType()` which just does `return this.inputs.potentialTemplates[0].targetType`

Reply:
done